src/SignalR/clients/csharp/Client.Core/src/Microsoft.AspNetCore.SignalR.Client.Core.csproj

```diff
@@ -27,10 +27,6 @@
     <Reference Include="System.Threading.Channels" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <Reference Include="Microsoft.Bcl.AsyncInterfaces" />
```

Comment:
Now relying on a transitive Microsoft.Bcl.AsyncInterfaces since it's now part of DependencyInjection.

- This was introduced as part of: dotnet/runtime#40189
- Prior to this we'd get version conflicts for our referenced Bcl.AsyncInterfaces.

/cc @dougbu @BrennanConroy @ViktorHofer @ericstj in case this added dependency was unintended.

Reply:
This is intentional.  The package was part of DI before, but it happened to be the same version you reference here (1.0.0).  We had to bring it back in 5.0.0 to solve an assembly versioning problem, so that means that there's a newer version.  This is just telling you to reference the newer version.

You could remove the reference, or update your reference (and add a subscription).

Reply:
It seems broken to remove this